rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -80,7 +80,6 @@ install -m 644 sd-app/* %{buildroot}/srv/salt/sd/sd-app/
 install -m 644 sd-proxy/* %{buildroot}/srv/salt/sd/sd-proxy/
 install -m 644 sd-whonix/* %{buildroot}/srv/salt/sd/sd-whonix/
 install -m 644 sd-workstation/* %{buildroot}/srv/salt/sd/sd-workstation/
-install -m 644 sys-firewall/* %{buildroot}/srv/salt/sd/sys-firewall/
```

Comment:
Can line 65 also be removed? `install -m 755 -d %{buildroot}/srv/salt/sd/sys-firewall`

Reply:
my bad - sure can